jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -823,6 +831,8 @@ protected void doStart() throws Exception
 
             // defers the calling of super.doStart()
             startContext();
+            
+            contextInitialized();
```

Comment:
Perhaps a comment here that this can be called earlier in the sequence by ServletHandler.